### PR TITLE
fix: hostd metrics

### PR DESCRIPTION
**Pull request by alexfreska**
- The metrics APIs now include wallet with balance and immatureBalance. https://github.com/SiaFoundation/hostd/issues/450
- The metrics API types now include all available fields.
- hostd: The contracts metrics now show active, rejected, failed, renewed, finalized, and successful.

### CODE REVIEW

**Comment by alexfreska** [comment]
* **#697** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/697?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alexfreska and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: libs/hostd-types/src/api.ts [hunk 1]
```diff
@@ -231,27 +231,26 @@ type Revenue = {
...
-type Contracts = {
-  pending: number
+type ContractMetrics = {
   active: number
   rejected: number
   failed: number
+  renewed: number
+  finalized: number
   successful: number
+  pending: number // TODO: remove?
```
Comment (alexfreska) [inline libs/hostd-types/src/api.ts:248]: @n8maninger pending is not shown here: https://github.com/SiaFoundation/hostd/issues/450, is this state gone?

Comment (n8maninger) [reply libs/hostd-types/src/api.ts:248]: Yeah, pending was removed from the metrics. It’s usually too short to matter

File: apps/hostd/contexts/metrics/index.tsx [hunk 2]
```diff
@@ -395,13 +396,28 @@ function useMetricsMain() {
       data,
       stats,
       config: {
-        enabledGraph: ['successful', 'active', 'pending', 'rejected', 'failed'],
-        enabledTip: ['successful', 'active', 'pending', 'rejected', 'failed'],
+        enabledGraph: [
+          'active',
```
Comment (alexfreska) [inline apps/hostd/contexts/metrics/index.tsx:400]: @n8maninger with the new breakdown, do these still work stacked and with a total per period? I've simply added them all to the existing graph:
Screenshot 2024-08-26 at 12 26 47 PM

**Review by ChrisSchinnerl — APPROVED** [review]

**Pull request merged**